### DISCUSSION TITLE
Event priority for example pointertap event before mousedown

### DISCUSSION THREAD

**Discussion by hvbnet** [pixijs/pixijs#7103] [Q&A]
Hi

I need to handle mouse click and mousedown/mousemove/mouseup to move Sprite from A to B (user can do it by drag and drop or click Sprite to select it and then click on location B to move)

but problem is that click event fire after mousedown and mouseup.

How to handle this issue?

**Comment by xzilja** [comment]
I don't think you can expect 'tap' to happen before 'up' event, not just in pixi, but in general js event scope. https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent

If you give a more detailed example of what you are trying to achieve i.e. what do you want to do with 'tap' event before 'down' / 'up'. I could try and figure this out with you.
